application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/ECChronos.java

```diff
@@ -124,6 +124,7 @@ private RepairConfiguration getRepairConfiguration(RepairProperties repairProper
                 .withParallelism(repairProperties.getRepairParallelism())
                 .withRepairWarningTime(repairProperties.getRepairAlarmWarnInMs(), TimeUnit.MILLISECONDS)
                 .withRepairErrorTime(repairProperties.getRepairAlarmErrorInMs(), TimeUnit.MILLISECONDS)
+                .withRepairLaziness(repairProperties.getRepairLaziness())
```

Comment:
Perhaps use the word UnwindRatio instead of Laziness

Reply:
Changed to unwindRatio across the code